libs/core/src/lib/tabs/tab-list.component.html

```diff
@@ -54,11 +57,41 @@
                     </ng-container>
                 </ng-container>
             </a>
-            <div fd-tab-process-icon *ngIf="mode === 'process' && i !== panelTabs.length - 1"></div>
+            <div fd-tab-process-icon *ngIf="mode === 'process' && !last"></div>
         </li>
 
-        <div *ngIf="tab.header" fd-tab-separator></div>
+        <div *ngIf="tab.isHeader" fd-tab-separator></div>
     </ng-container>
+
+    <li class="fd-tabs__item fd-tabs__item--overflow" #overflowTrigger *ngIf="_isCollapsed">
+        <a class="fd-tabs__link" [attr.tabindex]="0" [fdMenuTrigger]="menu">
```

Comment:
There is no way to trigger menu, by keyboard 
![image](https://user-images.githubusercontent.com/26483208/102492883-9b3f5300-4072-11eb-9e30-95bdcbcdc192.png)